studio/components/interfaces/Settings/Logs/Logs.utils.ts

```diff
@@ -397,6 +397,11 @@ export const fillTimeseries = (
   })
 
   const diff = maxDate.diff(minDate, truncation as dayjs.UnitType)
+  if (diff > 10000) {
+    throw new Error(
```

Comment:
is this error being handled at the moment if we're throwing one in this function? otherwise it's going to be incorrectly caught by sentry, and possibly show the client side error page to the user

Reply:
This is intentional, as charts shouldn't be rendering more than 10k data points and would indicate a bug.